modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/display/view.jsp

```diff
@@ -46,7 +48,7 @@ long formInstanceId = ddmFormDisplayContext.getFormInstanceId();
 			</clay:container-fluid>
 		</div>
 	</c:when>
-	<c:when test="<%= (ddmFormDisplayContext.isLimitToOneSubmissionPerUserEnabled() && !ddmFormDisplayContext.isLoggedUser()) || ddmFormDisplayContext.isRequireAuthentication() %>">
+	<c:when test="<%= (limitToOneSubmissionPerUser && !ddmFormDisplayContext.isLoggedUser()) || ddmFormDisplayContext.isRequireAuthentication() %>">
```

Comment:
Could you please sort the edited conditions in this JSP? Please [Brian's approach](https://github.com/brianchandotcom/liferay-portal/commit/303188c2471ecde39f5785274858e8d6c18ee5bd#commitcomment-62602093).

Reply:
Done!